### PR TITLE
DT-1121 Use default ingress

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

**Review by petergphillips — COMMENTED** [review]
does this work?  Normally should be
```
kubernetes.io/ingress.class: "nginx"
```

File: helm_deploy/probation-offender-search-indexer/templates/ingress.yaml [hunk 1]
```diff
@@ -11,7 +11,6 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
   annotations:
-    kubernetes.io/ingress.class: {{ .Release.Namespace }}
```
Comment (mattops) [inline helm_deploy/probation-offender-search-indexer/templates/ingress.yaml:14]: does this want to be ```kubernetes.io/ingress.class: nginx``` ?

Comment (mikehalmamoj) [reply helm_deploy/probation-offender-search-indexer/templates/ingress.yaml:14]: Looks to me like that is the default - CloudPlatform manually fixed the problems this morning by removing that annotation completely.

Comment (mikehalmamoj) [reply helm_deploy/probation-offender-search-indexer/templates/ingress.yaml:14]: However it won't hurt to be explicit and @petergphillips is on the case...

**Pull request closed without merging**